repose-aggregator/components/filters/valkyrie-authorization/src/main/resources/META-INF/schema/config/valkyrie-authorization.xsd

```diff
@@ -82,6 +82,22 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="pass-non-dedicated-tenant" type="xs:boolean" use="optional" default="false">
+            <xs:annotation>
+                <xs:documentation>
+                    <html:p>
+                        If enabled (true), the Valkyrie filter WILL NOT ensure that the tenant ID presented is a
+                        dedicated tenant ID (i.e., starts with "hybrid:"). If it is not, the Valkyrie filter will
+                        pass the request to the next filter in the filter chain.
```

Comment:
Feel free to tell me I'm being nit picky, but the use of "ensure" indicates that if the tenant ID isn't a dedicated tenant ID, the filter will _make it_ a dedicated tenant ID.  This is cleared up somewhat in the second sentence, but I'm still left slightly confused.  I think it would be clearer to say the filter will "verify" the tenant ID is for a dedicated tenant and if it isn't, immediately go to the next filter in the filter chain when this feature is enabled.